cmd/cctool/manifest.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+	"time"
+)
+
+type manifestConfig struct {
+	timeout time.Duration
+	pretty  bool
+}
+
+// Manifest is the subcommand for generating container manifests.
+func Manifest(cmd context.Context, cfg *commonConfig, args []string) error {
```

Comment:
var name `cmd` intentional?

Reply:
Yeah, it's keeping with the bad names I established in the report command.